src/styles.ts

```diff
@@ -1,8 +1,34 @@
 /* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
-import Yoga, {type YogaNode} from 'yoga-layout-prebuilt';
 import {type Boxes} from 'cli-boxes';
 import {type LiteralUnion} from 'type-fest';
 import {type ForegroundColorName} from 'chalk';
+import {
+	ALIGN_AUTO,
+	ALIGN_CENTER,
+	ALIGN_FLEX_END,
+	ALIGN_FLEX_START,
+	ALIGN_STRETCH,
+	DISPLAY_FLEX,
+	DISPLAY_NONE,
+	EDGE_BOTTOM,
+	EDGE_END,
+	EDGE_LEFT,
+	EDGE_RIGHT,
+	EDGE_TOP,
+	FLEX_DIRECTION_COLUMN,
+	FLEX_DIRECTION_COLUMN_REVERSE,
+	FLEX_DIRECTION_ROW,
+	FLEX_DIRECTION_ROW_REVERSE,
+	JUSTIFY_CENTER,
+	JUSTIFY_FLEX_END,
+	JUSTIFY_FLEX_START,
+	JUSTIFY_SPACE_AROUND,
+	JUSTIFY_SPACE_BETWEEN,
+	POSITION_TYPE_ABSOLUTE,
+	POSITION_TYPE_RELATIVE,
+	EDGE_START,
+	type YogaNode
+} from './yoga.js';
```

Comment:
This is a bit verbose. Maybe just `import * as Yoga ...`?

Reply:
Cool, applied